app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -132,19 +132,24 @@ const MeetingEditor = () => {
   );
 
   const fetchAndSetGroupMembers = async (groupId: number) => {
-    dispatch(fetchMemberships(groupId)).then((res) => {
-      setFetchedGroupIds((prevIds) => [...prevIds, groupId]);
-
-      const members = Object.values(res.payload.entities.users || {}).map(
-        (member) => ({
-          value: member.username,
-          label: member.fullName,
-          id: member.id,
-          groupId: groupId,
-        })
-      );
-      setInvitedGroupMembers((prevMembers) => [...prevMembers, ...members]);
-    });
+    dispatch(fetchMemberships({ groupId, propagateError: false }))
+      .then((res) => {
+        setFetchedGroupIds((prevIds) => [...prevIds, groupId]);
+
+        const members = Object.values(res.payload.entities.users || {}).map(
+          (member) => ({
+            value: member.username,
+            label: member.fullName,
+            id: member.id,
+            groupId: groupId,
+          })
+        );
+        setInvitedGroupMembers((prevMembers) => [...prevMembers, ...members]);
+      })
+      .catch(() => {
+        // User has no permission to load group members
+        setFetchedGroupIds((prevIds) => [...prevIds, groupId]);
+      });
```

Comment:
Hmm it's a fair point.. the reason I put it in the catch clause is that I didn't want the 403 to pop up in the console as it's sort of expected behavior. But with the finally it would still show.

I guess it might be better to ignore 403 but throw all other errors tho - as that's the only expected one.

Open to other suggestions to solve the problem if you have any